cmd/nerdctl/ps.go

```diff
@@ -106,7 +107,7 @@ func printContainers(ctx context.Context, cmd *cobra.Command, containers []conta
 	case "", "table":
 		w = tabwriter.NewWriter(os.Stdout, 4, 8, 4, ' ', 0)
 		if !quiet {
-			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES")
+			fmt.Fprintln(w, "CONTAINER ID\tIMAGE\tPLATFORM\tCOMMAND\tCREATED\tSTATUS\tPORTS\tNAMES\tRUNTIME")
```

Comment:
I think we should add `RUNTIME` field only when we have sufficient TTY width (and when we don't have TTY)


https://pkg.go.dev/golang.org/x/term#GetSize

Reply:
Or maybe only when `--no-trunc` is specified

Reply:
truncate means  "Don't truncate output" which isn't the right word to use here

Reply:
What about `--format wide`

Reply:
format means make output  according to some standards,
maybe -o wide is ok, like kubectl get node -o wide?

Reply:
but -o wide maybe conflict with  no-trunc

Reply:
I don't think we should have both `-o` and `--format`.

Reply:
maybe  --format wide is right word